### PR TITLE
[full-ci] Redesign Breadcrumb

### PR DESCRIPTION
**Pull request by lookacat**
## Related Issue
- Fixes https://github.com/owncloud/web/issues/6218
- Bumps ODS to v12.2.0 and resolves some minor visual glitches

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23013/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23013/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-23-01-04-38.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23013/alice-shares-file-to-brian-alice-2022-2-23-01-05-52.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23013/alice-shares-folder-with-file-to-brian-alice-2022-2-23-01-07-05.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/23053/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-file-to-brian-alice-2022-2-24-07-43-11.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-file-to-brian-brian-2022-2-24-07-43-17.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-folder-with-file-to-brian-alice-2022-2-24-07-44-38.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-folder-with-file-to-brian-brian-2022-2-24-07-44-45.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23053/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-24-07-43-07.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-creates-public-link-to-a-folder-alice-2022-2-24-07-44-13.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-file-to-brian-alice-2022-2-24-07-45-25.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-file-to-brian-brian-2022-2-24-07-45-33.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23053/alice-shares-folder-with-file-to-brian-alice-2022-2-24-07-46-45.zip 


</p></details>

**Review by pascalwengerter — APPROVED** [review]
LGTM once CI turns green

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/23054/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-file-to-brian-alice-2022-2-24-08-03-36.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-file-to-brian-brian-2022-2-24-08-03-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-folder-with-file-to-brian-alice-2022-2-24-08-04-58.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-folder-with-file-to-brian-brian-2022-2-24-08-05-04.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23054/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2022-2-24-08-04-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-creates-public-link-to-a-folder-alice-2022-2-24-08-05-50.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-file-to-brian-alice-2022-2-24-08-07-04.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23054/alice-shares-folder-with-file-to-brian-alice-2022-2-24-08-08-16.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/23055/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-file-to-brian-alice-2022-2-24-08-22-32.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-file-to-brian-brian-2022-2-24-08-22-36.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-folder-with-file-to-brian-alice-2022-2-24-08-23-49.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-folder-with-file-to-brian-brian-2022-2-24-08-23-54.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23055/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-file-to-brian-alice-2022-2-24-08-24-29.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-file-to-brian-brian-2022-2-24-08-24-39.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-folder-with-file-to-brian-alice-2022-2-24-08-26-03.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23055/alice-shares-folder-with-file-to-brian-brian-2022-2-24-08-26-11.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/23056/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-file-to-brian-alice-2022-2-24-08-58-36.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-file-to-brian-brian-2022-2-24-08-58-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-folder-with-file-to-brian-alice-2022-2-24-08-59-57.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-folder-with-file-to-brian-brian-2022-2-24-09-00-03.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/23056/10/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-file-to-brian-alice-2022-2-24-09-00-19.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-file-to-brian-brian-2022-2-24-09-00-28.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-folder-with-file-to-brian-alice-2022-2-24-09-01-51.zip 

#### npx playwright show-trace https://cache.owncloud.com/owncloud/web/tracing/23056/alice-shares-folder-with-file-to-brian-brian-2022-2-24-09-01-59.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Move</strong> https://drone.owncloud.com/owncloud/web/23058/21/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIMoveFilesFolders-moveFiles_feature-L139.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/23058/16/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesCopy-copy_feature-L120.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles4</strong> https://drone.owncloud.com/owncloud/web/23058/58/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFiles-breadcrumb_feature-L10.png
  

### webUIFiles-breadcrumb_feature-L21.png
  

### webUIFiles-breadcrumb_feature-L39.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAndUpload</strong> https://drone.owncloud.com/owncloud/web/23058/66/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:94

**Comment by ownclouders** [comment]
Results for <strong>oC10Files5</strong> https://drone.owncloud.com/owncloud/web/23058/20/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFiles-breadcrumb_feature-L10.png
  

### webUIFiles-breadcrumb_feature-L21.png
  

### webUIFiles-breadcrumb_feature-L39.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/23058/55/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesCopy-copy_feature-L120.png
  


</p></details>

**Comment by pascalwengerter** [comment]
Blocked by https://github.com/owncloud/owncloud-design-system/pull/1982 which with another ODS rc should turn acceptance tests green

File: tests/e2e/support/page/files/allFiles.ts
Comment (pascalwengerter) [inline tests/e2e/support/page/files/allFiles.ts:232]: @lookacat

**Comment by ownclouders** [comment]
Results for <strong>oC10Upload</strong> https://drone.owncloud.com/owncloud/web/23070/38/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIUpload-uploadEdgecases_feature-L99.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFavorites</strong> https://drone.owncloud.com/owncloud/web/23070/54/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFavorites-favoritesFile_feature-L58.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files4</strong> https://drone.owncloud.com/owncloud/web/23070/19/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesDetails-fileDetails_feature-L14.png
  

### webUIFilesDetails-fileDetails_feature-L173.png
  

### webUIFilesDetails-fileDetails_feature-L182.png
  

### webUIFilesDetails-fileDetails_feature-L27.png
  

### webUIFilesDetails-fileDetails_feature-L42.png
  

### webUIFilesDetails-fileDetails_feature-L57.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/23070/15/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares_feature-L50.png
  

### webUISharingAcceptSharesToRoot-acceptShares_feature-L31.png
  

### webUISharingAcceptSharesToRoot-acceptShares_feature-L50.png
  

### webUISharingAcceptSharesToRoot-acceptShares_feature-L89.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAndTrashbin</strong> https://drone.owncloud.com/owncloud/web/23070/33/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUITrashbinRestore-trashbinRestore_feature-L30.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10FileOperations</strong> https://drone.owncloud.com/owncloud/web/23070/14/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUICreateFilesFolders-createFile_feature-L13.png
  

### webUICreateFilesFolders-createFile_feature-L19.png
  

### webUICreateFilesFolders-createFile_feature-L25.png
  

### webUICreateFilesFolders-createFile_feature-L31.png
  

### webUIDeleteFilesFolders-deleteFilesFolders_feature-L253.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files2</strong> https://drone.owncloud.com/owncloud/web/23070/17/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesActionMenu-fileFolderActionMenu_feature-L14.png
  

### webUIFilesActionMenu-fileFolderActionMenu_feature-L37.png
  

### webUIFilesActionMenu-versions_feature-L15.png
  

### webUIFilesActionMenu-versions_feature-L26.png
  

### webUIFilesList-sort_feature-L31.png
  

### webUIFilesList-sort_feature-L51.png
  

### webUIFilesList-sort_feature-L72.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAutocompletion</strong> https://drone.owncloud.com/owncloud/web/23070/62/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAutocompletion-shareAutocompletion_feature-L56.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L66.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files3</strong> https://drone.owncloud.com/owncloud/web/23070/18/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUILogin-oauthLogin_feature-L22.png
  

### webUILogin-oauthLogin_feature-L41.png
  

### webUIRenameFiles-renameFiles_feature-L257.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Move</strong> https://drone.owncloud.com/owncloud/web/23070/21/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIMoveFilesFolders-moveFiles_feature-L105.png
  

### webUIMoveFilesFolders-moveFiles_feature-L114.png
  

### webUIMoveFilesFolders-moveFiles_feature-L12.png
  

### webUIMoveFilesFolders-moveFiles_feature-L20.png
  

### webUIMoveFilesFolders-moveFiles_feature-L43.png
  

### webUIMoveFilesFolders-moveFiles_feature-L51.png
  

### webUIMoveFilesFolders-moveFiles_feature-L79.png
  

### webUIMoveFilesFolders-moveFiles_feature-L81.png
  

### webUIMoveFilesFolders-moveFiles_feature-L82.png
  

### webUIMoveFilesFolders-moveFiles_feature-L85.png
  

### webUIMoveFilesFolders-moveFolders_feature-L12.png
  

### webUIMoveFilesFolders-moveFolders_feature-L20.png
  

### webUIMoveFilesFolders-moveFolders_feature-L33.png
  

### webUIMoveFilesFolders-moveFolders_feature-L41.png
  

### webUIMoveFilesFolders-moveFolders_feature-L66.png
  

### webUIMoveFilesFolders-moveFolders_feature-L68.png
  

### webUIMoveFilesFolders-moveFolders_feature-L69.png
  

### webUIMoveFilesFolders-moveFolders_feature-L88.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAndUpload</strong> https://drone.owncloud.com/owncloud/web/23070/66/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIUpload-uploadEdgecases_feature-L99.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Sharing1</strong> https://drone.owncloud.com/owncloud/web/23070/23/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L36.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L37.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L38.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L39.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L55.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L56.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L57.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L73.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L74.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L75.png
  

### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L76.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L101.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L115.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L128.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L141.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L154.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L168.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L56.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L66.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L76.png
  

### webUISharingAutocompletion-shareAutocompletion_feature-L87.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files5</strong> https://drone.owncloud.com/owncloud/web/23070/20/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFiles-breadcrumb_feature-L10.png
  

### webUIFiles-breadcrumb_feature-L39.png
  

### webUIFiles-breadcrumb_feature-L50.png
  

### webUIFiles-download_feature-L11.png
  

### webUIFiles-download_feature-L24.png
  

### webUIRenameFolders-renameFolders_feature-L106.png
  

### webUIRenameFolders-renameFolders_feature-L107.png
  

### webUIRenameFolders-renameFolders_feature-L111.png
  

### webUIRenameFolders-renameFolders_feature-L117.png
  

### webUIRenameFolders-renameFolders_feature-L122.png
  

### webUIRenameFolders-renameFolders_feature-L127.png
  

### webUIRenameFolders-renameFolders_feature-L133.png
  

### webUIRenameFolders-renameFolders_feature-L142.png
  

### webUIRenameFolders-renameFolders_feature-L161.png
  

### webUIRenameFolders-renameFolders_feature-L162.png
  

### webUIRenameFolders-renameFolders_feature-L163.png
  

### webUIRenameFolders-renameFolders_feature-L22.png
  

### webUIRenameFolders-renameFolders_feature-L23.png
  

### webUIRenameFolders-renameFolders_feature-L24.png
  

### webUIRenameFolders-renameFolders_feature-L25.png
  

### webUIRenameFolders-renameFolders_feature-L26.png
  

### webUIRenameFolders-renameFolders_feature-L38.png
  

### webUIRenameFolders-renameFolders_feature-L39.png
  

### webUIRenameFolders-renameFolders_feature-L40.png
  

### webUIRenameFolders-renameFolders_feature-L43.png
  

### webUIRenameFolders-renameFolders_feature-L58.png
  

### webUIRenameFolders-renameFolders_feature-L67.png
  

### webUIRenameFolders-renameFolders_feature-L79.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles3</strong> https://drone.owncloud.com/owncloud/web/23070/57/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUILogin-openidLogin_feature-L22.png
  

### webUILogin-openidLogin_feature-L29.png
  

### webUIRenameFiles-renameFiles_feature-L257.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles4</strong> https://drone.owncloud.com/owncloud/web/23070/58/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFiles-breadcrumb_feature-L10.png
  

### webUIFiles-breadcrumb_feature-L39.png
  

### webUIFiles-download_feature-L11.png
  

### webUIFiles-download_feature-L24.png
  

### webUIRenameFolders-renameFolders_feature-L106.png
  

### webUIRenameFolders-renameFolders_feature-L107.png
  

### webUIRenameFolders-renameFolders_feature-L117.png
  

### webUIRenameFolders-renameFolders_feature-L122.png
  

### webUIRenameFolders-renameFolders_feature-L127.png
  

### webUIRenameFolders-renameFolders_feature-L138.png
  

### webUIRenameFolders-renameFolders_feature-L142.png
  

### webUIRenameFolders-renameFolders_feature-L161.png
  

### webUIRenameFolders-renameFolders_feature-L162.png
  

### webUIRenameFolders-renameFolders_feature-L163.png
  

### webUIRenameFolders-renameFolders_feature-L22.png
  

### webUIRenameFolders-renameFolders_feature-L23.png
  

### webUIRenameFolders-renameFolders_feature-L24.png
  

### webUIRenameFolders-renameFolders_feature-L25.png
  

### webUIRenameFolders-renameFolders_feature-L26.png
  

### webUIRenameFolders-renameFolders_feature-L38.png
  

### webUIRenameFolders-renameFolders_feature-L39.png
  

### webUIRenameFolders-renameFolders_feature-L40.png
  

### webUIRenameFolders-renameFolders_feature-L43.png
  

### webUIRenameFolders-renameFolders_feature-L58.png
  

### webUIRenameFolders-renameFolders_feature-L67.png
  

### webUIRenameFolders-renameFolders_feature-L79.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/23070/71/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFavorites-favoritesFile_feature-L14.png
  

### webUIFavorites-unfavoriteFile_feature-L13.png
  

### webUIFilesCopy-copy_feature-L12.png
  

### webUIFilesCopy-copy_feature-L38.png
  

### webUIMoveFilesFolders-moveFiles_feature-L20.png
  

### webUIMoveFilesFolders-moveFiles_feature-L51.png
  

### webUIMoveFilesFolders-moveFolders_feature-L20.png
  

### webUIMoveFilesFolders-moveFolders_feature-L41.png
  

### webUISharingAcceptSharesToRoot-acceptShares_feature-L50.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/23070/69/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIMoveFilesFolders-moveFiles_feature-L105.png
  

### webUIMoveFilesFolders-moveFiles_feature-L114.png
  

### webUIMoveFilesFolders-moveFiles_feature-L12.png
  

### webUIMoveFilesFolders-moveFiles_feature-L20.png
  

### webUIMoveFilesFolders-moveFiles_feature-L43.png
  

### webUIMoveFilesFolders-moveFiles_feature-L51.png
  

### webUIMoveFilesFolders-moveFiles_feature-L79.png
  

### webUIMoveFilesFolders-moveFiles_feature-L81.png
  

### webUIMoveFilesFolders-moveFiles_feature-L82.png
  

### webUIMoveFilesFolders-moveFiles_feature-L85.png
  

### webUIMoveFilesFolders-moveFolders_feature-L12.png
  

### webUIMoveFilesFolders-moveFolders_feature-L20.png
  

### webUIMoveFilesFolders-moveFolders_feature-L33.png
  

### webUIMoveFilesFolders-moveFolders_feature-L41.png
  

### webUIMoveFilesFolders-moveFolders_feature-L66.png
  

### webUIMoveFilesFolders-moveFolders_feature-L68.png
  

### webUIMoveFilesFolders-moveFolders_feature-L69.png
  

### webUIMoveFilesFolders-moveFolders_feature-L88.png
  

### webUITrashbinRestore-trashbinRestore_feature-L30.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles2</strong> https://drone.owncloud.com/owncloud/web/23070/56/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesActionMenu-fileFolderActionMenu_feature-L14.png
  

### webUIFilesActionMenu-fileFolderActionMenu_feature-L37.png
  

### webUIFilesActionMenu-versions_feature-L15.png
  

### webUIFilesActionMenu-versions_feature-L26.png
  

### webUIFilesDetails-fileDetails_feature-L14.png
  

### webUIFilesDetails-fileDetails_feature-L173.png
  

### webUIFilesDetails-fileDetails_feature-L182.png
  

### webUIFilesDetails-fileDetails_feature-L27.png
  

### webUIFilesDetails-fileDetails_feature-L42.png
  

### webUIFilesDetails-fileDetails_feature-L57.png
  

### webUIFilesList-sort_feature-L31.png
  

### webUIFilesList-sort_feature-L51.png
  

### webUIFilesList-sort_feature-L72.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/23070/55/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUICreateFilesFolders-createFile_feature-L13.png
  

### webUICreateFilesFolders-createFile_feature-L19.png
  

### webUICreateFilesFolders-createFile_feature-L25.png
  

### webUICreateFilesFolders-createFile_feature-L31.png
  

### webUIDeleteFilesFolders-deleteFilesFolders_feature-L253.png
  

### webUIDeleteFilesFolders-deleteFilesFolders_feature-L77.png
  

### webUIFilesCopy-copy_feature-L12.png
  

### webUIFilesCopy-copy_feature-L131.png
  

### webUIFilesCopy-copy_feature-L142.png
  

### webUIFilesCopy-copy_feature-L28.png
  

### webUIFilesCopy-copy_feature-L38.png
  

### webUIFilesCopy-copy_feature-L70.png
  

### webUIFilesCopy-copy_feature-L71.png
  

### webUIFilesCopy-copy_feature-L72.png
  

### webUIFilesCopy-copy_feature-L73.png
  

### webUIFilesCopy-copy_feature-L76.png
  

### webUIMarkdownEditor-markdownFile_feature-L13.png
  

### webUIMarkdownEditor-markdownFile_feature-L21.png
  

### webUIMarkdownEditor-markdownFile_feature-L29.png
  

### webUIMarkdownEditor-markdownFile_feature-L37.png
  

### webUIMarkdownEditor-markdownFile_feature-L44.png
  

### webUIMarkdownEditor-markdownFile_feature-L50.png
  

### webUIMarkdownEditor-markdownFile_feature-L56.png
  

### webUIMarkdownEditor-markdownFile_feature-L70.png
  

### webUIMarkdownEditor-markdownFile_feature-L71.png
  

### webUIMarkdownEditor-markdownFile_feature-L80.png
  

### webUIMarkdownEditor-markdownFile_feature-L81.png
  

### webUIMarkdownEditor-markdownFile_feature-L82.png
  

### webUIMarkdownEditor-markdownFile_feature-L83.png
  

### webUIMarkdownEditor-markdownFile_feature-L84.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/23070/16/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFavorites-favoritesFile_feature-L111.png
  

### webUIFavorites-favoritesFile_feature-L130.png
  

### webUIFavorites-favoritesFile_feature-L14.png
  

### webUIFavorites-favoritesFile_feature-L30.png
  

### webUIFavorites-favoritesFile_feature-L46.png
  

### webUIFavorites-favoritesFile_feature-L58.png
  

### webUIFavorites-favoritesFile_feature-L65.png
  

### webUIFavorites-favoritesFile_feature-L74.png
  

### webUIFavorites-favoritesFile_feature-L81.png
  

### webUIFavorites-favoritesFile_feature-L86.png
  

### webUIFavorites-unfavoriteFile_feature-L104.png
  

### webUIFavorites-unfavoriteFile_feature-L13.png
  

### webUIFavorites-unfavoriteFile_feature-L34.png
  

### webUIFilesCopy-copyPrivateLinks_feature-L20.png
  

### webUIFilesCopy-copyPrivateLinks_feature-L21.png
  

### webUIFilesCopy-copy_feature-L12.png
  

### webUIFilesCopy-copy_feature-L131.png
  

### webUIFilesCopy-copy_feature-L142.png
  

### webUIFilesCopy-copy_feature-L28.png
  

### webUIFilesCopy-copy_feature-L38.png
  

### webUIFilesCopy-copy_feature-L70.png
  

### webUIFilesCopy-copy_feature-L71.png
  

### webUIFilesCopy-copy_feature-L72.png
  

### webUIFilesCopy-copy_feature-L73.png
  

### webUIFilesCopy-copy_feature-L76.png
  

### webUIMarkdownEditor-markdownFile_feature-L13.png
  

### webUIMarkdownEditor-markdownFile_feature-L21.png
  

### webUIMarkdownEditor-markdownFile_feature-L29.png
  

### webUIMarkdownEditor-markdownFile_feature-L37.png
  

### webUIMarkdownEditor-markdownFile_feature-L44.png
  

### webUIMarkdownEditor-markdownFile_feature-L50.png
  

### webUIMarkdownEditor-markdownFile_feature-L56.png
  

### webUIMarkdownEditor-markdownFile_feature-L70.png
  

### webUIMarkdownEditor-markdownFile_feature-L71.png
  

### webUIMarkdownEditor-markdownFile_feature-L80.png
  

### webUIMarkdownEditor-markdownFile_feature-L81.png
  

### webUIMarkdownEditor-markdownFile_feature-L82.png
  

### webUIMarkdownEditor-markdownFile_feature-L83.png
  

### webUIMarkdownEditor-markdownFile_feature-L84.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait2</strong> https://drone.owncloud.com/owncloud/web/23103/45/1
The following scenarios passed on retry:
- webUISharingExpirationDate/shareWithExpirationDate.feature:21

**Pull request merged**
